frontend/src/metabase/components/EntityMenu.info.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+
+import EntityMenu from 'metabase/components/EntityMenu'
+
+export const component = EntityMenu
+
+export const description = `
+    A menu with varios entity related options grouped by context.
+`
+
+const DemoAlignRight = ({ children }) =>
+    <div className="flex flex-full">
+            <div className="flex align-center ml-auto">
+                {children}
+            </div>
+        </div>
+
+export const examples = {
+    'Edit menu': (
+        <DemoAlignRight>
+            <EntityMenu
+                triggerIcon='pencil'
+                items={[
+                    { title: "Edit this question", icon: "editdocument", action: () => alert('Action type') },
```

Comment:
Instead of providing the individual items as an array of JS objects, I think it would be more idiomatic React to have a child component for them. An example is the "modernized" `<Select>` component which takes multiple `Option` as children, like in 

https://github.com/metabase/metabase/blob/master/frontend/src/metabase/admin/settings/components/widgets/CustomGeoJSONWidget.jsx#L210

This is not a hard blocker IMO but I like this API a little more and would be like to have consistency between different components that have similar parent-child model.

Reply:
Yeah I go back and forth on this. I actually kinda find the select pattern we have hard to remember how to use when I haven't encountered it in a while. 

There is something to be said for the clarity of importing EntiyMenuItem and using it as the child element at the same point as EntityMenu in the code I suppose. 

My only hesitation is there's more potential for edge cases or visual goofiness if you have people roll their own menu item internals in every place it's used, and one of the purposes of this menu pattern in the first place is to really unify our entity menu options, but I'm down to give it a shot.

Reply:
Yeah I'm fine with either approach.

> My only hesitation is there's more potential for edge cases or visual goofiness if you have people roll their own menu item internals in every place it's used.

True, it would potentially require some ugly React magic if you want to check that `this.props.children` only contains `EntityMenuItem`s ;)

Reply:
Cool. Let's maybe go with this as is for now and see how it feels as you're working with it in #6170. We can always adjust.